prince/util.py

```diff
@@ -133,7 +135,7 @@ def caller_name(skip=2):
 
     name = []
 
-    if config["print_module"]:
+    if print_module:
```

Comment:
I think this is a good example, where `config.print_module` would be much clearer.